.github/workflows/pr-reviews.yml

```diff
@@ -0,0 +1,22 @@
+# Request review on PRs without changing our codeowners file (which is stricter than review team)
+name: Request review on PRs
+
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+
+permissions:
+  pull-requests: write
+
+jobs:
+  request:
+    name: Request reviews on opened PRs
+    runs-on: ubuntu-latest
+    steps:
+      - name: Create PR review request
+        run: gh pr edit $PR_URL --add-reviewer @cosmos/sdk-core-review
+        env:
+          GH_TOKEN: ${{ secrets.PRBOT_PAT }}
```

Comment:
it does not seem to be available, as it seems was scoped to dependabot. we should scope it to this pr too.